book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
+
+  def renters
+    @rentals.map(&:person)
+  end
```

Comment:
- [ ] You don't need this method since you will be implementing the _**many-to-many**_ association between the `Book` and `Person` class as described in the `add_rental` method of this class. 👍